src/ui.message.js

```diff
@@ -75,7 +75,12 @@ var _ = Mavo.UI.Message = $.Class({
 	},
 
 	close: function(resolve) {
-		var promise = this.o.style.transition === "none" ? Promise.resolve() : $.transition(this.element, {opacity: 0});
+		if (this.options.style) {
+			var promise = this.options.style.transition === "none" ? Promise.resolve() : $.transition(this.element, {opacity: 0});
+		} 
+		else {
+			var promise = Promise.resolve();
+		}
```

Comment:
Why the change from a ternary to a multi-line conditional? 

Also, `promise` is not exactly a descriptive variable name. It's like calling a number variable `number` or a text variable `string`. Perhaps `closed` or something?